contracts/carrot-app/src/state.rs

```diff
@@ -62,8 +62,6 @@ impl AutocompoundRewardsConfig {
 #[cw_serde]
 pub struct PoolConfig {
     pub pool_id: u64,
-    pub token0: String,
```

Comment:
**Requires migrate fn**

Reply:
Added that, didn't think we planned to merge any contract